language/en/FLEET.php

```diff
@@ -1,109 +1,110 @@
-$LNG['sys_attacker_lostunits'] 				= "The attackers losses:";
-$LNG['sys_defender_lostunits'] 				= "The defenders losses:";
+$LNG['sys_attacker_lostunits'] 				= "Losses on the Attackers' side:";
+$LNG['sys_defender_lostunits'] 				= "Losses on the Defenders' side:";
 $LNG['sys_units']							= "units";
-$LNG['debree_field_1'] 						= "At the coordinates are now:";
+$LNG['debree_field_1'] 						= "At the coordinates appeared:";
 $LNG['debree_field_2']						= "debris.";
-$LNG['sys_moonproba'] 						= "Moon Chance: ";
-$LNG['sys_moonbuilt'] 						= "A moon formed at the planet %s [%d:%d:%d] !";
-$LNG['sys_attack_title']    				= "The following fleets face each other on: ";
+$LNG['sys_moonproba'] 						= "Chance of Forming a Moon: ";
+$LNG['sys_moonbuilt'] 						= "A Moon has formed at the planet %s [%d:%d:%d]!";
+$LNG['sys_attack_title']    				= "The following fleets clash on: ";
 $LNG['sys_attack_round']					= "Round";
-$LNG['sys_attack_attacker_pos'] 			= "Attacker";
+$LNG['sys_attack_attacker_pos'] 			= "Offense";
 $LNG['sys_attack_techologies'] 				= 'Weapons: %d %% Shields: %d %% Armor: %d %% ';
-$LNG['sys_attack_defender_pos'] 			= "Defender";
+$LNG['sys_attack_defender_pos'] 			= "Defense";
 $LNG['sys_ship_type'] 						= "Type";
 $LNG['sys_ship_count'] 						= "Amount";
 $LNG['sys_ship_weapon'] 					= "Firepower";
 $LNG['sys_ship_shield'] 					= "Shield";
 $LNG['sys_ship_armour'] 					= "Armour";
 $LNG['sys_destroyed'] 						= "Destroyed";
-$LNG['fleet_attack_1'] 						= "The attacking fleet fires with a power of";
-$LNG['fleet_attack_2']						= "at the defender. The shields of the defender absorb";
-$LNG['fleet_defs_1'] 						= "The defending fleet fires with a power of";
+$LNG['fleet_attack_1'] 						= "The attacking fleet fires with a total power of";
+$LNG['fleet_attack_2']						= "at the defenders. The shields of the defender absorb";
+$LNG['fleet_defs_1'] 						= "The defending fleet fires with a total power of";
 $LNG['fleet_defs_2']						= "at the attacker. The shields of the attacker absorb";
 $LNG['damage']								= "damage points.";
-$LNG['sys_attacker_won'] 					= "The attacker has won the battle!";
-$LNG['sys_defender_won'] 					= "The defender has won the battle!";
-$LNG['sys_both_won'] 						= "The fight ends in a draw! ";
-$LNG['sys_stealed_ressources'] 				= "He captures";
+$LNG['sys_attacker_won'] 					= "The Attacker has won the battle!";
+$LNG['sys_defender_won'] 					= "The Defender has won the battle!";
+$LNG['sys_both_won'] 						= "The fight ends in a draw!";
+$LNG['sys_stealed_ressources'] 				= "Thr Fleet captures";
 $LNG['sys_and']								= "and";
 $LNG['sys_mess_tower'] 						= "Fleet Headquarters";
-$LNG['sys_mess_attack_report'] 				= "Fight report";
+$LNG['sys_mess_attack_report'] 				= "Fight Report";
 $LNG['sys_spy_fleet'] 						= "Fleet";
 $LNG['sys_spy_defenses'] 					= "Defenses";
-$LNG['sys_mess_qg'] 						= "Spying department";
+$LNG['sys_mess_qg'] 						= "Central Department of Intelligence";
```

Comment:
"Central Department of Intelligence" as Spionageabteilung ? Sounds a bit weird to me, but sure we could still use it